frontend/src/pages/projects/screens/spawner/service.ts

```diff
@@ -30,13 +30,14 @@ import { fetchNotebookEnvVariables } from './environmentVariables/useNotebookEnv
 export const createPvcDataForNotebook = async (
   projectName: string,
   storageData: StorageData,
+  storageClassName?: string,
 ): Promise<{ volumes: Volume[]; volumeMounts: VolumeMount[] }> => {
   const { storageType } = storageData;
 
   const { volumes, volumeMounts } = getVolumesByStorageData(storageData);
 
   if (storageType === StorageType.NEW_PVC) {
-    const pvc = await createPvc(storageData.creating, projectName);
+    const pvc = await createPvc(storageData.creating, projectName, storageClassName);
```

Comment:
something I noticed: further down, at replaceRootVolumesForNotebook, createPVC does not pass the storageClassName https://github.com/opendatahub-io/odh-dashboard/blob/incubation/frontend/src/pages/projects/screens/spawner/service.ts#L72

Reply:
@andrewballantyne what do you think? Currently, incubation code does not have adding new storage / PVC with storageClassNamefrom the Storage Section (ManageStorageModal), only when adding new storage as part of "New Workbench" (SpawnerFooter). Would you advocate fixing f/storage before merging into main?